DICOMParser.py

```diff
@@ -55,6 +55,8 @@ def parse(self):
         with open(outputJSON) as jsonFile:
           measurementsJSON = json.load(jsonFile)
           self.readMeasurements(measurementsJSON)
+      else:
```

Comment:
Since template identifier is optional, can you also add the exception handling?

Reply:
I added an exception for that.